### PR TITLE
file-upload - Add file upload feature

### PR DESCRIPTION
**Pull request by filgaia**

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -7,33 +7,26 @@ import PropTypes from 'prop-types';
...
-import Button from '@material-ui/core/Button';
-import DeleteIcon from '@material-ui/icons/Delete';
+import SelectedCard from '../components/selectedCard/selectedCard';
+import UploadFile from '../components/uploadFile/uploadFile';
 
 class App extends Component {
 
     componentDidMount() {
-        this.props.getParticipantList();
+        // this.props.getParticipantList();
```
Comment (jhonandresmedina) [inline src/app/App.js]: If this change is intentional (as I think it's), please remove the componentDidMount hook

File: src/app/App.js [hunk 2]
```diff
@@ -45,30 +38,13 @@ class App extends Component {
...
-                                        color="secondary"
-                                        className="app__remove-button"
-                                        aria-label="delete"
-                                        onClick={() => this.removeParticipant(selectedParticipant.index)}>
-                                        Remove
-                                        <DeleteIcon />
-                                    </Button>
-                                </div>
-                            </div>
+                            <SelectedCard
```
Comment (jhonandresmedina) [inline src/app/App.js]: thank you for this change!

File: src/components/selectedCard/selectedCard.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+$participation-card-bg-color: #fdcfdc;
```
Comment (jhonandresmedina) [inline src/components/selectedCard/selectedCard.scss]: are we using this colors somewhere?

File: src/components/uploadFile/uploadFile.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+//material-components
+import Button from '@material-ui/core/Button';
+import CloudUpload from '@material-ui/icons/CloudUpload';
+
+class UploadFile extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = { file: null };
+        this.onFileChange = this.onFileChange.bind(this);
```
Comment (jhonandresmedina) [inline src/components/uploadFile/uploadFile.js]: this binders are not longer required, you can write your functions as:
`onFileChange = (e) => { ... }`

**Comment by filgaia** [comment]
Done!

**Pull request merged**